lib/rubocop/cop/rails/unique_validation_without_index.rb

```diff
@@ -31,7 +31,7 @@ class UniqueValidationWithoutIndex < Base
         RESTRICT_ON_SEND = %i[validates].freeze
 
         def on_send(node)
-          return unless uniqueness_part(node)
+          return unless uniqueness_part(node).truthy_literal?
```

Comment:
Can you update below?

```suggestion
          return if uniqueness_part(node).falsey_literal?
```

Reply:
Of course 👍